docs/user/ppl/functions/relevance.rst

```diff
@@ -352,6 +352,33 @@ Another example to show how to set custom values for the optional parameters::
     | 1    | The House at Pooh Corner | Alan Alexander Milne |
     +------+--------------------------+----------------------+
 
+
+HIGHLIGHT
+------------
+
+Description
+>>>>>>>>>>>
+
+``highlight(field_expression)``
+
+The highlight function maps to the highlight function used in search engine to return highlight fields for the given search.
+The syntax allows to specify the field in double quotes or single quotes or without any wrap.
```

Comment:
Backticks as well?

Reply:
Backticks can be used in identifiers. So they are functional but don't act as wildcard.